appveyor-windows.yml

```diff
@@ -80,6 +80,17 @@ test_script:
     - "docker system prune -a -f"
     - "pytest -vv tests/integration"
     - "pytest -vv tests/regression"
+    
+    # Change encoding to cp932
+    - ps: "Set-WinSystemLocale -SystemLocale ja-JP"
```

Comment:
This will add a lot of time into our tests. Can we not accomplish the same thing by create a specific test that will change the Locale for the duration of that test? 

What this gives us is a specific test that can be run anywhere (not just windows) that ensures if the system locale is different than utf8, things still work as expected. This will allow any developer to be able to verify locally, and not have to rely on specific setup/tests run on the ci/cd system. When this test will fail, we will know exactly what is broken vs thinking things are working, seeing ci/cd fail, and have to dig through failures which will look like all invokes are broken.

Reply:
Removed locale system change in favor of using unittest tests to mock open with different locale.